examples/listen-changes.rs

```diff
@@ -115,8 +114,6 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
         .start(|event| async move {
             match event {
                 FirestoreListenEvent::DocumentChange(ref doc_change) => {
-                    println!("Doc changed: {:?}", doc_change);
```

Comment:
Strange! It definitely gave me a compilation error before, because `gcloud_sdk::apis::google::firestore::v1::listen_response::ResponseType` didn't derive `Debug`. I even checked the source [here](https://github.com/abdolence/gcloud-sdk-rs/blob/ce009e99b9908b8c17cf5b4b8412920cfd52bcfd/gcloud-sdk/genproto/google.firestore.v1.rs#L1891). But looks like it compiles now, so I reverted those changes.